ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerDefinitionExpression/FeatureLayerDefinitionExpression.h

```diff
@@ -34,26 +35,33 @@ class FeatureLayerDefinitionExpression : public QQuickItem
   Q_OBJECT
 
   Q_PROPERTY(bool layerInitialized READ layerInitialized NOTIFY layerInitializedChanged)
+  Q_PROPERTY(int currentFeatureCount READ currentFeatureCount_ NOTIFY currentFeatureCountChanged)
 
 public:
   explicit FeatureLayerDefinitionExpression(QQuickItem* parent = nullptr);
   ~FeatureLayerDefinitionExpression() override;
 
   void componentComplete() override;
   static void init();
-  Q_INVOKABLE void setDefExpression(QString whereClause);
+  Q_INVOKABLE void setDefExpression(const QString& whereClause);
+  Q_INVOKABLE void setDisplayFilter(const QString& whereClause);
+  Q_INVOKABLE void updateFeatureCount();
```

Comment:
❓ Is this ever called directly from QML? If not, there is no need for `Q_INVOKABLE`.